src/routes/CreateWalletRoutes.tsx

```diff
@@ -39,12 +41,20 @@ const CreateWalletRoutes = () => {
   return (
     <WalletContextProvider>
       <Scrollbar>
-        <FloatingLogo />
-        <StepsContextProvider stepElements={createWalletSteps} baseUrl={'create'} />
-        <AppHeader />
+        <Container>
+          <AppHeader invisible />
```

Comment:
The header needs to be at the end of the DOM tree so that it renders above everything. This way, no magic with z-indexes is needed. That's why I had put it there!